### DISCUSSION TITLE
login to the app and extention

### DISCUSSION THREAD

**Discussion by lidoror** [dani-garcia/vaultwarden#2179] [General]
hi I am new and I just installed my vault warden but when I try to use the edge extension or the phone app I get  user not exist error what to do?

**Comment by RealOrangeOne** [comment]
Have you made sure the extension is pointing at your vaultwarden instance rather than bitwarden.com? 

https://bitwarden.com/help/article/change-client-environment/

**Reply by lidoror** [reply]
I changed it but now I am able to connect only via the pc in the same network how do I connect from a remote device?

**Reply by lidoror** [reply]
i saw in some video that i need to install certificate on the phone and laptop but i don't know how to download it

**Reply by RealOrangeOne** [reply]
That's a little out of scope for vaultwarden. Exactly how you want to expose vaultwarden to the internet (or not) is up to you. VPN, proxy, regular internet, port forward, up to you. I'm not sure what this "certificate" is, but exactly what that is and what/if you need to do something with it again depends on how you've exposed vaultwarden, and is again not vaultwarden specific so should be the same for any application, making finding support super simple.

**Reply by lidoror** [reply]
i exposed it with reverse proxy but its working only on my pc

**Reply by lidoror** [reply]
nevermind it's working now when my iPhone filled the credentials alone it get the user and password but when I typed the error disappeared.
